app/models/request.rb

```diff
@@ -0,0 +1,10 @@
+class Request < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :book
+
+  enum status:  [:pending, :accepted, :rejected]
+
+  def has_passed?
```

Comment:
`has` is not used (and not recommended as well on Ruby), is better if you only name it `passed?`.